### PR TITLE
done view list of article

### PR DESCRIPTION
**Pull request by jd4545**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,12 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <DisplayArticles />
```
Comment (popatre) [inline src/App.js]: You might find you have to put this in a home route, moving forward. Cross that bridge when you come to it

File: src/components/display-articles.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    //Setting ARTICLE state:
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles().then((articles) => {
+          setArticles(articles);
+        });
+      }, []);
+
+    // ARTICLES on HOME page
```
Comment (popatre) [inline src/components/display-articles.jsx]: - [ ] Is loading - for user feedback during the api request
- [ ] Error handling - incase the request has a problem - little error message/component that displays instead

**Review by popatre — CHANGES_REQUESTED** [review]
Just have a look at those two small things, then merge.

**Comment by jd4545** [comment]
Added isLoading for articles page, and error for incorrect routes

**Pull request merged**